### PR TITLE
[YUNIKORN-1006] Address QU 30, 40, 50

### PR DESCRIPTION
**Pull request by yangwwei**
This PR addresses the QU 30, 40, 50 in the ASF project maturity model doc.

QU30

The project provides a well-documented, secure and private channel to report security issues, along with a documented way of responding to them.

QU40

The project puts a high priority on backwards compatibility and aims to document any incompatible changes and provide tools and documentation to help users transition to new features.

CO50

The project documents how contributors can earn more rights such as commit access or decision power, and applies these principles consistently.

### CODE REVIEW

**Comment by yangwwei** [comment]
@chenya-zhang , @wilfred-s please help to review, thanks

File: src/pages/community/get_involved.md [hunk 1]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+  - Integrate with other projects, extend the use cases
+
+Contributor or non-contributor can both make their paths to committer, the community deeply
+appreciates both code or non-code contributions.
+
+## Become a PPMC member
+
+PPMC represents for Podling Project Management Committee, this committee is responsible for project
+management and governance, ensures the project can be operated under [Apache Way](https://www.apache.org/theapacheway/).
+The committee makes decisions based on the PPMC members' votes. A PPMC member has the authority to cast a binding cast
```
Comment (chenya-zhang) [inline src/pages/community/get_involved.md:109]:
```suggestion
The committee makes decisions based on the PPMC members' votes. A PPMC member has the authority to cast a binding
```

File: src/pages/community/get_involved.md [hunk 2]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+a binding vote on various things, such as project releases, adding new committer or PPMC member, etc.
```
Comment (chenya-zhang) [inline src/pages/community/get_involved.md:110]:
```suggestion
on various things, such as project releases, adding new committer or PPMC member, etc.
```

**Review by chenya-zhang — APPROVED** [review]
Thanks Weiwei! I just saw you already sent a PR so I assigned the ticket back to you. ;)

btw. For QU40: The project puts a high priority on backwards compatibility and aims to document any incompatible changes and provide tools and documentation to help users transition to new features.

=> Should we update our release process (maybe also publish our release process to website when mature) and add a "Backward Incompatibility" section in our future release notes?

File: src/pages/community/get_involved.md [hunk 3]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
 
 - Calendar: [Google Calendar](https://calendar.google.com/calendar/b/3/r/week/2020/3/26?cid=YXBhY2hlLnl1bmlrb3JuQGdtYWlsLmNvbQ) :point_left:
 
+## Become a Committer
+
+Committers are the community members who have the write access to the project's repositories, i.e
+they can modify the code by themselves and accept others contributions to all YuniKorn repos.
+There is no strict rules about the qualification of a candidate, the PPMC votes for a candidate
```
Comment (yuchaoran2011) [inline src/pages/community/get_involved.md:83]:
```suggestion
There is no strict rules about the qualification of a candidate. The PPMC votes for a candidate
```

File: src/pages/community/get_involved.md [hunk 4]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+based on various considerations:
+
+- Code contributions
+  - Consistent code contributions
+  - Optimize the CI/CD pipeline
+  - Help the code reviews
```
Comment (yuchaoran2011) [inline src/pages/community/get_involved.md:89]:
```suggestion
  - Help with code reviews
```

File: src/pages/community/get_involved.md [hunk 5]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+  - Test and verify release candidates
+  - Performance tuning and related tools development
+  - Propose improvement proposals
+
+- Non-code contributions
+  - Involvement in the community activities, such as meetings, meetups, etc.
+  - Provide feedback, report issues, roadmap discussions
```
Comment (yuchaoran2011) [inline src/pages/community/get_involved.md:96]:
```suggestion
  - Provide feedback, report issues, and participate in roadmap discussions
```

File: src/pages/community/get_involved.md [hunk 6]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+  - Improve project documentation
+  - Help the project adoption
+  - Integrate with other projects, extend the use cases
+
+Contributor or non-contributor can both make their paths to committer, the community deeply
+appreciates both code or non-code contributions.
+
+## Become a PPMC member
+
+PPMC represents for Podling Project Management Committee, this committee is responsible for project
```
Comment (yuchaoran2011) [inline src/pages/community/get_involved.md:107]:
```suggestion
PPMC stands for the Podling Project Management Committee. It is responsible for project
```

File: src/pages/community/get_involved.md [hunk 7]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+management and governance, ensures the project can be operated under [Apache Way](https://www.apache.org/theapacheway/).
```
Comment (yuchaoran2011) [inline src/pages/community/get_involved.md:108]:
```suggestion
management and governance and ensures that the project can be operated under the [Apache Way](https://www.apache.org/theapacheway/).
```

File: src/pages/community/get_involved.md [hunk 8]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+The committee makes decisions based on the PPMC members' votes. A PPMC member has the authority to cast a binding cast
+a binding vote on various things, such as project releases, adding new committer or PPMC member, etc.
+
+A contributor must become a committer first before being a PPMC member.
```
Comment (yuchaoran2011) [inline src/pages/community/get_involved.md:112]:
```suggestion
A contributor must become a committer first before becoming a PPMC member.
```

File: src/pages/community/get_involved.md [hunk 9]
```diff
@@ -76,6 +76,44 @@ import TabItem from '@theme/TabItem';
...
+There is no strict rules of when a committer will be qualified for adding to PPMC. The management
```
Comment (yuchaoran2011) [inline src/pages/community/get_involved.md:113]:
```suggestion
There is no strict rules of when a committer will be qualified for being added to the PPMC. The management
```

File: src/pages/community/how_to_contribute.md [hunk 10]
```diff
@@ -96,6 +96,12 @@ The commit message must include the JIRA in the first line and should include a
 A commit will automatically close the PR.
 The JIRA will not be closed automatically.
 
+## Report a security issue
+YuniKorn community cares deeply about the security and actively support for any security issues as
```
Comment (yuchaoran2011) [inline src/pages/community/how_to_contribute.md:100]:
```suggestion
YuniKorn community cares deeply about the security and actively addresses any security issues as
```

File: src/pages/community/how_to_contribute.md [hunk 11]
```diff
@@ -96,6 +96,12 @@ The commit message must include the JIRA in the first line and should include a
...
+the top priority. The community uses its private mailing list (private@yunikorn.apache.org) to discuss
+and track security related issues. If any security issue found, please email to the private
```
Comment (yuchaoran2011) [inline src/pages/community/how_to_contribute.md:102]:
```suggestion
and track security related issues. If any security issue found, please email the private
```

**Review by yuchaoran2011 — CHANGES_REQUESTED** [review]
Overall looks great. Some nits on typos and grammar

**Comment by yangwwei** [comment]
> Thanks Weiwei! I just saw you already sent a PR so I assigned the ticket back to you. ;)
> 
> btw. For QU40: The project puts a high priority on backwards compatibility and aims to document any incompatible changes and provide tools and documentation to help users transition to new features.
> 
> => Should we update our release process (maybe also publish our release process to website when mature) and add a "Backward Incompatibility" section in our future release notes?

yes, make sense. I will add that today as well. Thank you!!

File: src/pages/community/get_involved.md (hunk 1 above)
Comment (yangwwei) [reply src/pages/community/get_involved.md]: Done

File: src/pages/community/get_involved.md (hunk 2 above)
Comment (yangwwei) [reply src/pages/community/get_involved.md]: Done

File: src/pages/community/get_involved.md (hunk 3 above)
Comment (yangwwei) [reply src/pages/community/get_involved.md]: Done

File: src/pages/community/get_involved.md (hunk 6 above)
Comment (yangwwei) [reply src/pages/community/get_involved.md]: done

File: src/pages/community/get_involved.md (hunk 7 above)
Comment (yangwwei) [reply src/pages/community/get_involved.md]: DOne

File: src/pages/community/get_involved.md (hunk 9 above)
Comment (yangwwei) [reply src/pages/community/get_involved.md]: done

File: src/pages/community/how_to_contribute.md (hunk 11 above)
Comment (yangwwei) [reply src/pages/community/how_to_contribute.md]: done

File: src/pages/community/how_to_contribute.md (hunk 10 above)
Comment (yangwwei) [reply src/pages/community/how_to_contribute.md]: done

File: src/pages/community/get_involved.md (hunk 8 above)
Comment (yangwwei) [reply src/pages/community/get_involved.md]: done

**Review by yuchaoran2011 — APPROVED** [review]

**Pull request merged**